File: python-sdk/src/formanceapi/models/shared/getholdresponse.py
import dataclasses
from ..shared import expandeddebithold as shared_expandeddebithold
from dataclasses_json import dataclass_json
from formanceapi import utils


@dataclass_json
@dataclasses.dataclass
class GetHoldResponse:
    data: shared_expandeddebithold.ExpandedDebitHold = dataclasses.field(metadata={'dataclasses_json': { 'letter_case': utils.field_name('data') }})
    

File: ts-sdk/src/sdk/models/shared/getholdresponse.ts
import { SpeakeasyBase, SpeakeasyMetadata } from "../../../internal/utils";
import { ExpandedDebitHold } from "./expandeddebithold";


export class GetHoldResponse extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "json, name=data" })
  data: ExpandedDebitHold;
}